docs/resources/connection.md

```diff
@@ -35,7 +35,7 @@ resource "aci_connection" "conn2" {
 ## Argument Reference
 
 - `l4_l7_service_graph_template_dn` - (Required) Distinguished name of parent L4-L7 Service Graph Template object.
-- `name` - (Required) Name of object connection.
+- `name` - (Required) Name of object connection. The valid connection name format is `CONX`, where X is a number starting with 0. Type: String.
```

Comment:
Should we add type behind the other attributes also now we add to this one?

Reply:
added